spec/fixtures/proofing/lexis_nexis/ddp/successful_response.json

```diff
@@ -5,5 +5,6 @@
   "risk_rating": "trusted",
   "summary_risk_score": "-6",
   "tmx_risk_rating": "neutral",
-  "fraudpoint.score": "500"
+  "fraudpoint.score": "500",
+  "first_name": "WARNING! YOU SHOULD NEVER SEE THIS PII FIELD IN THE LOGS"
```

Comment:
That's a nice way to test!

Reply:
yes.  if you have suggestion on how to make that message more clear they are welcome.  essentially if we see that in our log files we know there is a bug